client/clientBean.go

```diff
@@ -256,7 +273,7 @@ func NewDNSYarpcDispatcherProvider(logger log.Logger, interval time.Duration) Di
 	}
 }
 
-func (p *dnsDispatcherProvider) Get(serviceName string, address string) (*yarpc.Dispatcher, error) {
+func (p *dnsDispatcherProvider) Get(serviceName string, address string, options *DispatcherOptions) (*yarpc.Dispatcher, error) {
```

Comment:
Do I need to run something to get the latest mock for this?  Quoting you @longquanzheng 

```There are two implementation, one is mock. The real one is dnsYarpcDispatcherProvider.```